packages/itwinui-react/src/core/Buttons/DropdownButton/DropdownButton.tsx

```diff
@@ -28,7 +28,7 @@ export type DropdownButtonProps = {
    * Use 'borderless' to hide outline.
    * @default 'default'
    */
-  styleType?: 'default' | 'borderless';
+  styleType?: 'default' | 'borderless' | 'high-visibility';
```

Comment:
Should we add CTA button too?

Reply:
lets wait until someone asks? imo the CTA button should not be used anywhere